FileOperationsCases.cs

```diff
@@ -0,0 +1,81 @@
+﻿using HelloWorld.Models;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json.Schema;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+
+namespace HelloWorld
+{
+     public class FileOperationsCases
+    {
+        public void WritingJsonToFile()
+        {
+            HomeActivity homeActivity = new HomeActivity("Ironing", 2, "Saturday");
```

Comment:
create HomeActivity instance in your demo class and pass the value as a parameter to the method